src/workerd/io/worker.c++

```diff
@@ -1884,7 +1875,8 @@ void Worker::handleLog(jsg::Lock& js,
     auto colors =
         COLOR_MODE == ColorMode::ENABLED || (COLOR_MODE == ColorMode::ENABLED_IF_TTY && tty);
 
-    auto inspectModule = resolveNodeInspectModule(js);
+    static constexpr auto kSpecifier = "node-internal:internal_inspect"_kj;
```

Comment:
3-4 months ago I think either @fhanau or @mikea mentioned that static constexpr is unnecessary and constexpr is sufficient. I don't remember why tho...

Reply:
Yeah, there's a clang-tidy lint that triggers for it (can't have a storage class for constexpr). Plain constexpr should work well